examples/feature_demo/lightmap.py

```diff
@@ -40,7 +40,7 @@
 canvas = WgpuCanvas(size=(1200, 400), title="lightmap")
 renderer = gfx.renderers.WgpuRenderer(canvas)
 
-meshes = gfx.load_meshes(model_dir / "lightmap" / "scene.gltf")
+meshes = gfx.load_mesh(model_dir / "lightmap" / "scene.gltf")
```

Comment:
🤔 I'm a bit confused. The method is singular but it returns a list. Would it not be better to deprecate `load_mesh()` instead?

@Korijn you made a comment related to this earlier. Do you mean we should deprecate/remove one of these, or specifically `load_meshes()`?

Also I found out (after writing the above) that there's `trimesh.load_mesh()`. If we want to align our API a bit with trimesh here, I think that's a valid argument.

Reply:
I would prefer `load_mesh` and I would prefer it to return a single object. However it seems there are various file formats that return multiple objects. I think it's preferred that return types don't vary based on what's in the file, so always returning a list is an okay compromise to me. So:

`list_of_meshes = load_mesh(filename)`

Reply:
My argument is mostly, if `load_mesh()` returns a list of meshes (which may contain just one mesh), is not not better to call it `load_meshes()`? 

> We could add a keyword argument to control the return type:

It's actually shorter for a user to add `[0]`  after the call :)

Reply:
> My argument is mostly, if `load_mesh()` returns a list of meshes (which may contain just one mesh), is not not better to call it `load_meshes()`?
> 
> > We could add a keyword argument to control the return type:
> 
> It's actually shorter for a user to add `[0]` after the call :)

To be honest, I'm okay with either at this point, it feels like bikeshedding. What matters to me is that we offer the functionality in an unsurprising way, and that the API is predictable. Those things are at odds here.

load_mesh is more consistent (predictable) since other load functions are also singular, but load_meshes is less surprising because of the return type being a list.

So both are suboptimal and I don't really mind which one we pick. We'll have to document them well and then that will be that. :)

Reply:
🤷 I agree. Let's move on :)